packages/vector_graphics_compiler/bin/vector_graphics_compiler.dart

```diff
@@ -63,7 +68,7 @@ Future<void> main(List<String> args) async {
     }
   }
 
-  if (results['optimize-masks'] == true) {
+  if (results['optimize-masks'] == true || results['optimize-clips'] == true) {
```

Comment:
nit: main is getting kind of long, consider moving these to helper methods that are helpfully named e.g. `loadPathOpsIfNeeded(results)` or something like that.

Reply:
fixed.